_apps/dep.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Dep
+description: A dependency manager for your Pull Requests.
+slug: dep
+screenshots:
+- https://github.com/ahmed-taj/dep/raw/master/docs/assets/comment.png
+- https://github.com/ahmed-taj/dep/raw/master/docs/assets/status.png
+authors:
+- ahmed-taj
+repository: ahmed-taj/dep
+host: https://dep.now.sh
+---
+
+<p align="center">
+  <img src="https://github.com/ahmed-taj/dep/raw/master/docs/assets/logo.png" width="256" height="256" alt="bot logo">
+</p>
+
+A dependency manager for your Pull Requests.
+
+## Usage
+
+1. Browse to [GitHub Apps - dep][apps]
+2. Accept the permissions
+3. Allow access to repositories
+
+If you want to update a pull request dependencies write:
+
+![comment-screenshot][]
+
+On the pull request, a status check from `dep` will appear:
+
+![status-check-screenshot][]
+
+For best results, enable branch protection (in the repository's settings) and require the `dep` status check to pass before merging:
+
+![branch-protection-screenshot][]
+
+[apps]: https://github.com/apps/dep
+[status-check-screenshot]: https://github.com/ahmed-taj/dep/raw/master/docs/assets/status.png
+[comment-screenshot]: https://github.com/ahmed-taj/dep/raw/master/docs/assets/comment.png
+[branch-protection-screenshot]: https://github.com/ahmed-taj/dep/raw/master/docs/assets/settings.png
+
+## Contributors
+
+Thanks goes to these wonderful people ([emoji key](https://github.com/kentcdodds/all-contributors#emoji-key)):
+
+<!-- ALL-CONTRIBUTORS-LIST:START - Do not remove or modify this section -->
+| [<img src="https://avatars1.githubusercontent.com/u/12673605?v=4" width="100px;"/><br /><sub><b>Ahmed T. Ali</b></sub>](https://ahmed.sd)<br />[📝](https://medium.com/ahmed-t-ali/lets-build-a-github-pro-bot-5e155cec395f "Blogposts") [💻](https://github.com/ahmed-taj/dep/commits?author=ahmed-taj "Code") [📖](https://github.com/ahmed-taj/dep/commits?author=ahmed-taj "Documentation") [⚠️](https://github.com/ahmed-taj/dep/commits?author=ahmed-taj "Tests") | [<img src="https://avatars1.githubusercontent.com/u/10660468?v=4" width="100px;"/><br /><sub><b>Jason Etcovitch</b></sub>](https://github.com/JasonEtco)<br />[💬](https://twitter.com/JasonEtco "Answering Questions") [🤔](https://github.com/probot/ideas/issues/1 "Ideas, Planning, & Feedback") |
+| :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: |
+<!-- ALL-CONTRIBUTORS-LIST:END -->
```

Comment:
The rendering on this section doesn't quite look right. I don't have time to dig into it right now, but would you be willing to see if you can tweak this so it renders properly on the website?

<img width="794" alt="dep___probot" src="https://user-images.githubusercontent.com/173/32752455-dcd0693a-c88e-11e7-8d50-6ca63aa653e7.png">

Reply:
Will give it a try